workspace.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
is this file local from vs code @lbassil ?

Reply:
Not needed. I deleted it.